### DISCUSSION TITLE
Add FEC doesn't work

### DISCUSSION THREAD

**Discussion by AlexRSmith95** [Mr-MIBonk/M.I.B._More-Incredible-Bash#303] [Q&A]
Hi all,

I have followed the guides closely and patched my unit with the AiO.

I have then tried to add the FECs using MIB however they won't add to my unit. Please could somebody help me identify where I'm going wrong?

I will attach my log for information.
Thanks!!

[MU1325-HHA-07309.07.2070090036.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12251454/MU1325-HHA-07309.07.2070090036.log)

**Comment by AlexRSmith95** [comment]
> Hi all,
> 
> I have followed the guides closely and patched my unit with the AiO.
> 
> I have then tried to add the FECs using MIB however they won't add to my unit. Please could somebody help me identify where I'm going wrong?
> 
> I will attach my log for information. Thanks!!
> 
> [MU1325-HHA-07309.07.2070090036.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12251454/MU1325-HHA-07309.07.2070090036.log)

Specifically, I am looking for the lifetime map FEC as I want to update from 22/23 to 23/24.

**Reply by Falconer999** [reply]
I'm facing the same "issue" at the moment /sad

**Comment by Falconer999** [comment]
> Hi all,
> 
> I have followed the guides closely and patched my unit with the AiO.
> 
> I have then tried to add the FECs using MIB however they won't add to my unit. Please could somebody help me identify where I'm going wrong?
> 
> I will attach my log for information. Thanks!!
> 
> [MU1325-HHA-07309.07.2070090036.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/12251454/MU1325-HHA-07309.07.2070090036.log)

@AlexRSmith95 ok I manged to add 023000EE into my unit. what I did was
put M.I.B 3.6 into a SD Card
grabbed the patch file (unzipped) for my unit from https://mib.mibsolution.one/ and put it into MIB 3.6 patch folder
for.ex I have FW: MHI2Q_ER_AUG22_P5092
it need to looks like patch/MHI2Q_ER_AUG22_P5092_MU1329_PATCH/patched file
under the same folder, you also need to put the matching addFEC.txt for your car along with ExceptionList.txt
then run the "add FEC list" under AIO first,
when you see the 023000EE has been added to the list
then run the AIO

I had to run AIO twice for some reason to have 023000EE added to my unit, but it worked after.

so good luck!
